src/driver/connection.rs

```diff
@@ -127,15 +127,6 @@ impl Connection {
 
 #[pymethods]
 impl Connection {
-    #[must_use]
```

Comment:
It feels unnecessary.
__aenter__ and __aexit__ here the main methods, as I see

Reply:
I'm not sure, It needs to be checked)